docs/UsersGuide/source/SRWAppOverview.rst

```diff
@@ -184,6 +186,8 @@ parameters in the ``set_predef_grid_params`` script.
    | RRFS_CONUS_3km       | ESG grid          | lambert_conformal              |
    +----------------------+-------------------+--------------------------------+
 
+NOTE DOM 20220227 would it be good to mention the number of horizontal grid points NX x NY in this table/section? Or any other information (min/max lat/lon etc)? or a plot of the three domains? Or provide a link to section 6.1?
+
 Case-specific Configuration
```

Comment:
Since the user can create their own grid, I don't think there's a reason to point them to the pre-defined grids here.

Reply:
This section is just meant to provide a general overview of the utilities.

Reply:
I added a pointer to chpt 6. That hopefully suffices.